client/extensions/woocommerce/woocommerce-services/state/service-settings/values/actions.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import _ from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import * as api from 'woocommerce/woocommerce-services/api';
+import coerceFormValues from 'woocommerce/woocommerce-services/lib/utils/coerce-values';
+import * as FormActions from '../actions';
+import * as NoticeActions from 'state/notices/actions';
+import getFormErrors, { EMPTY_ERROR } from '../selectors/errors';
+
+export const UPDATE_FIELD = 'UPDATE_FIELD';
+export const REMOVE_FIELD = 'REMOVE_FIELD';
+export const ADD_ARRAY_FIELD_ITEM = 'ADD_ARRAY_FIELD_ITEM';
+
+export const updateField = ( path, value ) => ( {
+	type: UPDATE_FIELD,
+	path,
+	value,
+} );
+
+export const removeField = ( path ) => ( {
+	type: REMOVE_FIELD,
+	path,
+} );
+
+export const addArrayFieldItem = ( path, item ) => ( {
+	type: ADD_ARRAY_FIELD_ITEM,
+	path,
+	item,
+} );
+
+export const submit = ( schema, silent ) => ( dispatch, getState, { methodId, instanceId, formType } ) => {
+	silent = ( true === silent );
+
+	const setIsSaving = ( value ) => dispatch( FormActions.setFormProperty( 'isSaving', value ) );
+
+	const setSuccess = ( value ) => {
+		dispatch( FormActions.setFormProperty( 'success', value ) );
+		if ( ! silent && true === value ) {
+			dispatch( NoticeActions.successNotice( translate( 'Your changes have been saved.' ), {
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Your changes have been saved.', { context: 'voce settings api'} )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/copy/wcs-data-friven-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).